README.md

```diff
@@ -1,2 +1,74 @@
 # StaticMaps
-Static Map renderer using mapnik
+Static Map rendering service using mapnik. Post GeoJSON, get back a map rendered on a background.
```

Comment:
This is fine, but when you check it in I'll propose a new PR.

I would organize into headings and make the first sentence complete and forceful